File: jwtprac/src/main/java/com/member/jwtprac/controller/JoinController.java
package com.member.jwtprac.controller;

import com.member.jwtprac.dto.JoinDTO;
import com.member.jwtprac.service.JoinService;
import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequiredArgsConstructor
public class JoinController {

    private final JoinService joinService;

    @GetMapping("/join")
    public String joinProcess(JoinDTO joinDTO) {

        joinService.joinProcess(joinDTO);

        return "ok";
    }
}

File: jwtKotlin/src/main/kotlin/com/member/jwtkotlin/controller/JoinController.kt
package com.member.jwtkotlin.controller

import com.member.jwtkotlin.dto.JoinDTO
import com.member.jwtkotlin.service.JoinService
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.ResponseBody

@Controller
@ResponseBody
class JoinController(joinService: JoinService) {
    private val joinService: JoinService = joinService

    @PostMapping("/join")
    fun joinProcess(joinDTO: JoinDTO): String {
        System.out.println(joinDTO.username)
        joinService.joinProcess(joinDTO)

        return "ok"
    }
}